### ISSUE TITLE
Long passage selection

### ISSUE THREAD

**Issue by cbocionek** [annefried/discanno#163]
When scrolling multiple times to try to select a long passage (>20 lines), the passage in question cannot be annotated as the rest of the document is selected instead.

**Comment by JeannedArk** [comment]
Is it the same behavior like here described: https://github.com/annefried/discanno/issues/155 ?

**Comment by cbocionek** [comment]
No, this happens regardless what length the document has. The text does not disappear, either.
